plugins/module_utils/data_set.py

```diff
@@ -335,6 +387,234 @@ def data_set_member_exists(name):
             return False
         return True
 
+    @staticmethod
+    def data_set_shared_members(src, dest):
+        """Checks for the existence of members from a source data set in
+        a destination data set.
+
+        Arguments:
+            src (str) -- The source data set name. The name can contain a wildcard pattern.
+            dest (str) -- The destination data set name.
+
+        Returns:
+            bool -- If at least one of the members in src exists in dest.
+        """
+        src_members = datasets.list_members(src)
+
+        for member in src_members:
+            if DataSet.data_set_member_exists("{0}({1})".format(dest, member)):
+                return True
+
+        return False
+
+    @staticmethod
+    def get_member_name_from_file(file_name):
```

Comment:
Just checking to make sure you really don't need 8 in any of your cases using this code. I know qualifiers don't need 8 but just wanted to double check. 
Maybe also a minor tweak the description,  "Creates a member name for a partitioned data set by taking UP TOO the  first 8 characters from a filename without its file extension" .

Reply:
Thanks for the resolution